docs/tasks/inject-data-application/podpreset.md

```diff
@@ -20,18 +20,50 @@ You can get an overview of PodPresets at
 This is a simple example to show how a Pod spec is modified by the Pod
 Preset.
 
-**User submitted pod spec:**
+{% include code.html language="yaml" file="podpreset-preset.yaml" ghlink="/docs/tasks/inject-data-application/podpreset-preset.yaml" %}
+
+Create the PodPreset:
+
+```shell
+kubectl create -f https://k8s.io/docs/tasks/inject-data-application/podpreset-preset.yaml
+```
+
+Examine the created PodPreset:
+
+```shell
+kubectl get podpreset
+NAME             AGE
+allow-database   1m
+```
+
+The new PodPreset will act upon any pod that has label `role: frontend`.
 
 {% include code.html language="yaml" file="podpreset-pod.yaml" ghlink="/docs/tasks/inject-data-application/podpreset-pod.yaml" %}
 
-**Example Pod Preset:**
+Create a pod:
 
-{% include code.html language="yaml" file="podpreset-preset.yaml" ghlink="/docs/tasks/inject-data-application/podpreset-preset.yaml" %}
+```shell
+kubectl create -f https://k8s.io/docs/tasks/inject-data-application/podpreset-pod.yaml
+```
+
+List the running Pods:
+
+```shell
+kubectl get pods
```

Comment:
Please add `$` before the command, so that viewer can distinguish between command and it's output, it's okay to not have `$` if we are just showing command to run without it's output.